jobs/config.json

```diff
@@ -2357,6 +2375,23 @@
       "UNKNOWN"
     ]
   },
+  "ci-kubernetes-e2e-gce-gci-qa-serial-m62": {
+    "args": [
+      "--check-leaked-resources",
+      "--env-file=jobs/platform/gce.env",
+      "--extract=gci/gci-62",
+      "--gcp-project=e2e-gce-gci-qa-serial-m62",
+      "--gcp-zone=us-central1-f",
+      "--provider=gce",
+      "--test_args=--ginkgo.focus=\\[Serial\\]|\\[Disruptive\\] --ginkgo.skip=\\[Flaky\\]|\\[Feature:.+\\] --minStartupPods=8",
+      "--timeout=300m",
+      "--ginkgo-parallel=25"
```

Comment:
This is the *serial* suite though

Reply:
Ahh. Sorry. Too saturated to get even copy-paste right :(
Removed.